contracts/0.8.9/Burner.sol

```diff
@@ -54,10 +54,10 @@ contract Burner is IBurner, AccessControlEnumerable {
 
     error ErrorAppAuthLidoFailed();
     error ErrorDirectETHTransfer();
-    error ZeroRecoveryAmount();
-    error StETHRecoveryWrongFunc();
-    error ZeroBurnAmount();
-    error NotEnoughExcessStETH();
+    error ErrorZeroRecoveryAmount();
```

Comment:
I don't thing that this prefix is meaningful. And also, there is a ton of `Zero...` errors across the codebase.
We need to change them all or revert these changes. I'd rather revert. If you have something reverted it's obviously a error, we don't need to repeat it everywhere.

Reply:
refactoring here and there, thx!